developers/integrations/llm-frameworks/llamaindex/LlamaIndex.md

```diff
@@ -0,0 +1,44 @@
+---
+title: LlamaIndex
+sidebar_position: 5
+image: og/docs/llamaindex.jpg
+---
+
+[LlamaIndex](https://www.llamaindex.ai/) is a framework for building large language model (LLM) applications. 
+
+## LlamaIndex and Weaviate
+Weaviate is a [supported vector store](https://docs.llamaindex.ai/en/stable/api_reference/storage/vector_store/weaviate/) in LlamaIndex. You need a running Weaviate instance to build the vector store. You can create it with:
+
+```python
+vector_store = WeaviateVectorStore(weaviate_client=client, index_name="LlamaIndex")
+```
+
+## Our Resources 
+The resources are broken into two categories: 
+1. [**Hands on Learning**](#hands-on-learning): Build your technical understanding with end-to-end tutorials.
+
+2. [**Read and Listen**](#read-and-listen): Develop your conceptual understanding of these technologies.
+
+### Hands on Learning
+
+| Topic | Description | Resource | 
+| --- | --- | --- |
+| Data Loaders in LlamaIndex | Learn how to load data into Weaviate using LlamaIndex, and connect LlamaIndex to an existing Weaviate cluster. | [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/llm-frameworks/llamaindex/data-loaders-episode1/episode1.ipynb) |
```

Comment:
Learn how to load data into Weaviate using LlamaIndex, and how to connect LlamaIndex to an